src/components/playlist/PlaylistHeading.tsx

```diff
@@ -80,24 +95,17 @@ export function PlaylistHeading({
         fontSize={{ base: "2xl", sm: "3xl", md: "4xl", lg: "5xl" }}
       >
         {editTitle ? (
-          <form onSubmit={submitHandlerTitle}>
-            <InputGroup
-              colorScheme={titleInvalid ? "red" : "brand"}
-              size={"lg"}
-            >
-              <Input
-                placeholder="Playlist Title"
-                autoFocus
-                isInvalid={titleInvalid}
-                defaultValue={title}
-                onChange={(e) =>
-                  setTitleInvalid(!isValid(e.currentTarget.value, 1, 70))
-                }
-                enterKeyHint="done"
-              />
-              {titleInvalid && <InputRightAddon color="red" children="1~70" />}
-            </InputGroup>
-          </form>
+          <InputGroup colorScheme={titleInvalid ? "red" : "brand"} size={"lg"}>
+            <Input
+              placeholder="Playlist Title"
+              autoFocus
+              isInvalid={titleInvalid}
+              defaultValue={title}
+              onChange={(e) => titleChangeHandler(e.currentTarget.value)}
+              enterKeyHint="done"
```

Comment:
Enter button doesn't do anything anymore. We can maybe just remove it and only use save changes button